File: packages/php/examples/background_removal.php
<?php

declare(strict_types=1);

require dirname(__DIR__) . '/vendor/autoload.php';

use AILabTools\AILabClient;
use AILabTools\Generated\Params\CutoutUniversalBackgroundRemovalParams;
use AILabTools\Http\FileInput;

$apiKey = getenv('AILAB_API_KEY');
if ($apiKey === false || $apiKey === '') {
    throw new RuntimeException('Set AILAB_API_KEY before running this example.');
}

$client = new AILabClient(apiKey: $apiKey);
$result = $client->background->remove(
    new CutoutUniversalBackgroundRemovalParams(
        image: FileInput::fromPath(dirname(__DIR__, 3) . '/image.jpg'),
    ),
);

echo $result->data?->imageUrl . PHP_EOL;

File: packages/rust/examples/background_removal.rs
use ailabtools::generated::params::CutoutUniversalBackgroundRemovalParams;
use ailabtools::{Client, FileInput, Result};

#[tokio::main]
async fn main() -> Result<()> {
    let client = Client::new(std::env::var("AILAB_API_KEY").expect("AILAB_API_KEY is required"))?;
    let params = CutoutUniversalBackgroundRemovalParams {
        image: FileInput::from_path("image.jpg", Some("image/jpeg"))?,
        return_form: Some("whiteBK".to_string()),
    };
    let response = client.background.remove(params).await?;
    println!("request_id={:?} data={:?}", response.request_id, response.data);
    Ok(())
}
